practice_app/pages/api/getTranslation.js

```diff
@@ -0,0 +1,46 @@
+const axios = require('axios')
+const { checkTranslationResponse } = require('./getTranslationModule')
+
+export default async (req, res) => {
+    // req object contains whatever information we can get from the request that comes from the client (from the browser)
+    // res object is used tow write our reponse
+    const { query, method } = req
+
+    switch (method) {
+        case 'GET':
+            {
+                console.log(query)
+                let target = query.countryCode == null ? '' : query.countryCode
+                target = target.toLowerCase()
+                const text = query.text == null ? '' : query.text
+                let sourceLang = query.sl == null ? '' : query.sl
```

Comment:
Agreed with Mehdi, I think if countryCode, text or source language is null, you should nor proceed further. You can directly return an error.

Reply:
Fixed.